docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -136,6 +151,27 @@ export default function MintlistPage() {
                 </div>
               </div>
             </div>
+            <div>
+              <h2>NFTs</h2>
+              {user &&
+                data.mintlist.authority === user.address &&
+                data.mintlist.mint_infos.length <
+                  data.mintlist.num_nfts_total && (
```

Comment:
It might be nice to pull this out into a variable rather than doing it all in the render.